python/mxnet/contrib/quantization.py

```diff
@@ -285,6 +285,10 @@ def _get_optimal_threshold(arr, num_bins=8001, num_quantized_bins=255):
     max_val = np.max(arr)
     th = max(abs(min_val), abs(max_val))
 
+    if min_val >= 0 and quantized_dtype in ['auto', 'uint8']:
```

Comment:
how do the numbers change here for accuracy after this fix: https://cwiki.apache.org/confluence/display/MXNET/MXNet+Graph+Optimization+and+Quantization+based+on+subgraph+and+MKL-DNN

Reply:
The numbers collected in that link are based on naive calibration method, so they won't be changed after this fix.